assembly/compliance-tool/src/main/java/org/eclipse/sw360/antenna/frontend/compliancetool/sw360/status/SW360StatusReporterParameters.java

```diff
@@ -0,0 +1,139 @@
+package org.eclipse.sw360.antenna.frontend.compliancetool.sw360.status;
+
+import org.eclipse.sw360.antenna.frontend.compliancetool.main.AntennaComplianceToolOptions;
+import org.eclipse.sw360.antenna.frontend.stub.cli.AbstractAntennaCLIOptions;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Objects;
+import java.util.Optional;
+import java.util.Set;
+
+/**
+ * Class responsible for parsing status reporter parameters
+ * and creating the {@code InfoRequest} the status reporter
+ * uses.
+ */
+class SW360StatusReporterParameters {
```

Comment:
The class name is a bit misleading. Maybe something like ParametersHelper or ParameterParser?